platforms/bukkit/common/src/main/resources/plugin.yml

```diff
@@ -6,3 +6,4 @@ author: dfsek
 website: "@WIKI@"
 api-version: "1.13"
 description: "@DESCRIPTION@"
+folia-supported: true
```

Comment:
minor issue: this file should have a newline at the end (unsure if it does, bc I'm on gh mobile)

Reply:
Looks like it does not